src/main/kotlin/no/digdir/fdk/searchservice/model/CommonDataClasses.kt

```diff
@@ -33,6 +33,76 @@ data class ReferenceDataCode(
   val prefLabel: LocalizedStrings?,
 )
 
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+data class ConformsTo(
+  val uri: String?,
+  val prefLabel: LocalizedStrings?,
+)
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+data class InSeries(
+  val id: String?,
+  val uri: String?,
+  val title: LocalizedStrings?,
+)
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+data class Subject(
+  val identifier: String?,
+  val uri: String?,
+  val prefLabel: LocalizedStrings?,
+  val definition: LocalizedStrings?
+)
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+data class Reference(
+  val referenceType: ReferenceDataCode?,
+  val source: String?,
+)
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+data class Relation(
```

Comment:
flytt eller kopier denne inn til SearchObject, for å tydeliggjøre at det bare er den som skal bruke den og endre type til å være enum